grails-app/migrations/2018-10-01-v500-job-label-optional.groovy

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog = {
+    changeSet(author: "fho", id: "20181001-label-1") {
+        dropNotNullConstraint(columnDataType: "varchar(255)", tableName: "job", columnName: "label")
+    }
+    changeSet(author: "fho", id: "20181008-label-1") {
```

Comment:
Sorry to start bothering, but can you drop a short comment why this is necessary? It's not quite obvious from the code ;)